he.lproj/Localizable.strings

```diff
@@ -161,73 +161,73 @@
 
 /* Broadcasts v2.0 */
 
-"ERROR_ALERT_TITLE" = "Error: %@";
+"ERROR_ALERT_TITLE" = "שגיאה: %@";
 
-"LIBRARY_STATE_UNAUTHENTICATED" = "Not Logged In";
-"LIBRARY_STATE_QUOTA_EXCEEDED" = "Out of Space";
+"LIBRARY_STATE_UNAUTHENTICATED" = "לא מחובר";
+"LIBRARY_STATE_QUOTA_EXCEEDED" = "שטח האחסון מלא";
 
-"SIDEBAR_CATEGORY_DIRECTORY" = "Browse";
-"SIDEBAR_TITLE_DIRECTORY" = "Directory";
+"SIDEBAR_CATEGORY_DIRECTORY" = "עיון";
+"SIDEBAR_TITLE_DIRECTORY" = "סייר התחנות";
 
-"EDIT_STATION_WINDOW_INTERNET_RADIO_PLACEHOLDER" = "Internet Radio";
+"EDIT_STATION_WINDOW_INTERNET_RADIO_PLACEHOLDER" = "רדיו אינטרנט";
 
-"STATION_METADATA_NAME_PLACEHOLDER" = "Untitled Station";
+"STATION_METADATA_NAME_PLACEHOLDER" = "תחנה חדשה";
 
-"WELCOME_BUTTON_CONTINUE" = "Continue";
-"WELCOME_BUTTON_GETSTARTED" = "Get Started";
+"WELCOME_BUTTON_CONTINUE" = "המשך";
+"WELCOME_BUTTON_GETSTARTED" = "התחל";
 
-"WELCOME_NEWFEATURES_TITLE" = "New Features";
+"WELCOME_NEWFEATURES_TITLE" = "תכונות חדשות";
 
-"WELCOME_NEWFEATURES_2_DESIGN" = "Modern look & feel";
-"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "Sleep Timer";
-"WELCOME_NEWFEATURES_2_AIRPLAY" = "AirPlay toolbar controls";
-"WELCOME_NEWFEATURES_2_ALERTS" = "Less-intrusive alerts";
-"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Native pixel scaling";
-"WELCOME_NEWFEATURES_2_APPLESILICON" = "Supports Apple Silicon";
+"WELCOME_NEWFEATURES_2_DESIGN" = "מראה ותחושה מודרניים";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "שעון עצר לשינה";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "שליטה על AirPlay";
+"WELCOME_NEWFEATURES_2_ALERTS" = "התראות פחות מפריעות";
```

Comment:
I am not too satisfied about this string... That said, I do find it challenging to come up with something better.
How about: "התראות יותר מעודנות"? Just throwing it out there, your version works as well.